psi4/src/psi4/libmints/onebody.cc

```diff
@@ -80,6 +84,71 @@ static void transform1e_2(int am, SphericalTransformIter &sti, double *s, double
 }
 }  // namespace
 
+//! Form shell pair information for one electron routines, which don't use the shell pair information
+std::vector<std::pair<int, int>> build_shell_pair_list_no_spdata(std::shared_ptr<BasisSet> bs1,
+                                                                 std::shared_ptr<BasisSet> bs2, double threshold) {
+    const auto nsh1 = bs1->nshell();
+    const auto nsh2 = bs2->nshell();
+    const auto bs1_equiv_bs2 = (bs1 == bs2);
+    auto nthreads = Process::environment.get_n_threads();
+
+    // construct the 2-electron repulsion integrals engine
+    using libint2::Engine;
+    std::vector<Engine> engines;
+    engines.reserve(nthreads);
+    engines.emplace_back(libint2::Operator::overlap, std::max(bs1->max_nprimitive(), bs2->max_nprimitive()),
+                         std::max(bs1->max_am(), bs2->max_am()), 0);
+    for (size_t i = 1; i != nthreads; ++i) {
+        engines.push_back(engines[0]);
+    }
+    std::vector<std::vector<std::pair<int, int>>> threads_sp_list(nthreads);
+
+    threshold *= threshold;
+    ///ACS TODO make sure we set the threshold appropriately
```

Comment:
Nope - I've left a comment.  I've set everything up so that we can simply turn on shell pair screening later on, but that will inevitably break test cases.  The current behavior (no screening) is exactly what the current master does.